README.md

```diff
@@ -3,43 +3,89 @@
 [![Travis branch](https://img.shields.io/travis/aragon/radspec/master.svg?style=flat-square)](https://travis-ci.org/aragon/radspec)
 [![Coveralls github branch](https://img.shields.io/coveralls/github/aragon/radspec/master.svg?style=flat-square)](https://coveralls.io/github/aragon/radspec)
 
-Radspec is a safe alternative to Ethereum's natspec[[?](#aside-why-is-natspec-unsafe)].
+Radspec is a language specification and interpreter for dynamic expressions in Ethereum's NatSpec.
+
+This allows smart contact developers to show improved function documentation to end users.
+
+![Screen Shot 2019-04-04 at 10 44 19 AM](https://user-images.githubusercontent.com/382183/55565167-5cfbd780-56c7-11e9-8ca8-24c727e54ab5.png)
 
 ## Features
 
-- **External calls**: Radspec can perform calls to external contracts
-- **Safe**: Radspec has no DOM access at all.
-- **Simple**: Even though radspec requires you to inline types for external calls, the syntax is very familiar and readable (it looks like Flow).
-- **Compatible**: Most natspec comments that already exist are also compatible ith Radspec.
+- **Expressive**: Show relevant details to Smart contract end-users at the time they make transactions.
+- **External calls**: Radspec can query other contracts.
+- **Safe**: Radspec has no access to DOM.
+- **Compatible**: Most existing NatSpec dynamic expressions are compatible with Radspec.
+
+## Introduction & quick start
+
+Radspec supports any contract programming language, such as Solidity or Vyper because radspec works on the compiled JSON ABI. Here is an example using Solidity.
+
+```solidity
+pragma solidity ^0.5.0;
+
+contract Tree {
+    /// @notice Set the tree age to `numYears` years
+    function setAge(uint256 numYears) external {
+        // set the age into storage
+    }
+}
+```
+
+Notice the *dynamic expression* documentation for the `age` function. When presented to the end user, this will render based on the inputs provided by the user. For example, if the end user is calling the contract with an input of 10 years, this is specified to render as:
```

Comment:
```suggestion
Notice the *dynamic expression* documentation for the `setAge` function. When presented to the end user, this will render based on the inputs provided by the user. For example, if the end user is calling the contract with an input of 10 years, this is specified to render as:
```